utah-design-system-website/src/react/components/websiteContent/library/components/widgetsIndicators/badges/BadgesExampleProps.jsx

```diff
@@ -0,0 +1,25 @@
+import { Form, TextArea, TextInput } from '@utahdts/utah-design-system';
+
+/** @typedef {import('utah-design-system-website').BadgesExamplePropsShape} BadgesExamplePropsShape */
+
+/**
+ * @param {object} props
+ * @param {import('use-immer').Updater<{props: BadgesExamplePropsShape}>} props.setState
+ * @param {object} props.state
+ * @returns {import('react').JSX.Element}
+ */
+
+export function BadgesExampleProps({ setState, state }) {
+  return (
+    <Form
+      className="form--stacked"
+      // @ts-ignore
```

Comment:
The props.state param has a type of `object`. Change that type to:
``` * @param {{props: BadgesExamplePropsShape}} props.state```
and the @ts-ignore is no longer needed